kolibri/plugins/coach/assets/src/modules/examCreation/actions.js

```diff
@@ -0,0 +1,183 @@
+import pickBy from 'lodash/pickBy';
+import uniq from 'lodash/uniq';
+import unionBy from 'lodash/unionBy';
+import union from 'lodash/union';
+import { ContentNodeResource, ContentNodeSearchResource } from 'kolibri.resources';
+import { createTranslator } from 'kolibri.utils.i18n';
+import { getContentNodeThumbnail } from 'kolibri.utils.contentNode';
+import router from 'kolibri.coreVue.router';
+import { ContentNodeKinds } from 'kolibri.coreVue.vuex.constants';
+import { PageNames } from '../../constants';
+import { _createExam } from '../shared/exams';
+
+const snackbarTranslator = createTranslator('ExamCreateSnackbarTexts', {
+  newExamCreated: 'New exam created',
+});
+
+export function resetExamCreationState(store) {
```

Comment:
If these one-mutation type actions aren't used in too many components, I like using the mapMutations helper in the components, e.g. `...mapMutations('examCreation', { setTitle: 'SET_TITLE' })`